GooglePayment/src/main/java/com/braintreepayments/api/models/ReadyForGooglePayRequest.java

```diff
@@ -0,0 +1,32 @@
+package com.braintreepayments.api.models;
+
+import com.braintreepayments.api.BraintreeFragment;
+import com.braintreepayments.api.GooglePayment;
+import com.braintreepayments.api.interfaces.BraintreeResponseListener;
+
+/**
+ * Optional parameters to use when checking whether Google Payment API is supported and set up on the device.
+ */
+public class ReadyForGooglePayRequest {
```

Comment:
Do we want to me consistent with the terminology of `GooglePayment` as opposed to `GooglePay`?

Reply:
Sure!